configure.ac

```diff
@@ -277,8 +272,6 @@ AS_CASE([${host_os}],
 dnl ====================================================================================
 dnl  Now use the information from the checking stage.
 
-AC_DEFINE_UNQUOTED([OS_IS_WIN32], [${os_is_win32}], [Set to 1 if compiling for Win32])
```

Comment:
This is significantly better than what you want to replace it with.

Reply:
Reverted